deploy/backplane/mtsre/ocs-consumer/01-mtsre-ocs-consumer-cr-admins.ClusterRole.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-mtsre-ocs-consumer-cr-admins
+rules:
+  - apiGroups:
+    - "ocs.openshift.io"
+    resources:
+      - "storageclassclaims"
```

Comment:
nit: whitespace is inconsistent

Reply:
Whoops! Thanks for the catch, let me fix this up!